src/components/shared/Util.ts

```diff
@@ -1,6 +1,21 @@
 import * as am4core from '@amcharts/amcharts4/core';
 import * as am4charts from '@amcharts/amcharts4/charts';
 
+const BRACKET_LEVEL_PREDICATES: { [key: number]: Function } = {
+  0: (player: any) => player, // no filter selected, default value
+  1: (player: any) => player.level < 20,
+  2: (player: any) => player.level > 20 && player.level < 30,
+  3: (player: any) => player.level > 30 && player.level < 40,
+  4: (player: any) => player.level > 40 && player.level < 50,
+  5: (player: any) => player.level > 50 && player.level < 60,
+  6: (player: any) => player.level > 60 && player.level < 70,
+  7: (player: any) => player.level > 70,
+};
+
```

Comment:
the level 20 is never read, 30 neither and so on
use <= or >= in some cases

```suggestion
const BRACKET_LEVEL_PREDICATES: { [key: number]: Function } = {
  0: (player: any) => player, // no filter selected, default value
  1: (player: any) => player.level < 20,
  2: (player: any) => player.level >= 20 && player.level < 30,
  3: (player: any) => player.level >= 30 && player.level < 40,
  4: (player: any) => player.level >= 40 && player.level < 50,
  5: (player: any) => player.level >= 50 && player.level < 60,
  6: (player: any) => player.level >= 60 && player.level < 70,
  7: (player: any) => player.level >= 70,
};

```

Reply:
we could even put a bracket with only 80 :P

Reply:
Done. Please check them now.